tox.ini

```diff
@@ -14,12 +14,13 @@ DJANGO =
     master: djangomaster
 
 [testenv]
-commands = python -Wall -m unittest discover tests
+commands = python -m runtests
 setenv =
     PYTHONDONTWRITEBYTECODE=1
-
+    DATABASE_URL=mysql://root:@127.0.0.1:3306/atomiqdb
 deps =
     .
+    -rrequirements-dev.txt
```

Comment:
no that's how it is. idk why

Reply:
A single `-` indicates a one character flag, whereas `--` indicates multiple characters. The single char makes it unambiguous to parse and enables some shortcuts. With single dash the following are equivalent within group:

```
-t -i
-ti
```

```
-r requirements-dev.txt
-rrequirements-dev.txt
```

Reply:
Having said that `-r requirements-dev.txt` should be much preferred since it's clearer and avoids comments like this thread. Not blocking though obvs.